src/App.jsx

```diff
@@ -1,15 +1,16 @@
-
-import React from 'react';
-import Overview from './components/overview/Overview.jsx';
-import Reviews from './components/reviews/Reviews.jsx';
-import Questions from './components/questions/Questions.jsx';
-import Related from './components/related/Related.jsx';
+import React from "react";
+import axios from "axios";
+import Overview from "./components/overview/Overview.jsx";
+import Reviews from "./components/reviews/Reviews.jsx";
+import Questions from "./components/questions/Questions.jsx";
+import Related from "./components/related/Related.jsx";
+import API_KEY from "./config.js";
 
 class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      productId: 11 // TODO - find out what this should actually be.
+      productId: 11, // TODO - find out what this should actually be.
```

Comment:
Do we want to make the trailing comma standard for the codebase?

Reply:
I would vote against the trailing commas. Also, perhaps we remove the comment TODO since we now populate the productId after the component mounts.

Reply:
We can also change the 11 to null so that we don't get confused what 11 is.